### PR TITLE
Augmentations not existent in Multianimal setup - fix

### PR DESCRIPTION
**Pull request by aristotelis-economides**
The imgaug augmentations introduced in [this](https://github.com/DeepLabCut/DeepLabCut/pull/1166/commits/8544f3179caa6abab23b16beb3913aa99756c313) pull request were not part of the multianimal augmentation code, I added them. 

It runs properly as far as I can tell, but DLC in general does not "complain" when settings in the config files are not being parsed. (i.e. the "clahe" augmentation was part of my pose_cfg.yaml, but I think it was not used at all since the function was not implemented in the code?)  

Also there is a minor typo fix in the recipes/nn.py docs.

### CODE REVIEW

**Review by AlexEMG — APPROVED** [review]

File: deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py [hunk 1]
```diff
@@ -120,6 +121,80 @@ def build_augmentation_pipeline(self, height=None, width=None, apply_prob=0.5):
...
+            cfg_cnt[aug] = aug_val
+            if aug_val:
+                cfg_cnt[aug + "ratio"] = cfg_cnt.get(aug + "ratio", 0.1)
+
+        convolution_aug = ["sharpen", "emboss", "edge"]
+        for aug in convolution_aug:
+            aug_val = cfg_cnv.get(aug, False)
+            cfg_cnv[aug] = aug_val
+            if aug_val:
+                cfg_cnv[aug + "ratio"] = cfg_cnv.get(aug + "ratio", 0.1)
```
Comment (jeylau) [inline deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py:149]: What do you say about the following?
```python
contrast_aug = ["histeq", "clahe", "gamma", "sigmoid", "log", "linear"]
convolution_aug = ["sharpen", "emboss", "edge"]
for aug in contrast_aug + convolution_aug:
    ...
```

Comment (AlexEMG) [reply deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py:149]: Yeah, fine

Comment (aristotelis-economides) [reply deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py:149]: Didn't even know that python supports + in a for statement! Crazy language. Pushed this now :)

Comment (jeylau) [reply deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py:149]: Yeah, lists get concatenated first ☺️

**Review by jeylau — APPROVED** [review]

File: deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py (hunk 1 above)
Comment (aristotelis-economides) [reply deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py:149]: Tests failing now though? What did I do 🤯

Comment (aristotelis-economides) [reply deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py:149]: fixed..

Comment (jeylau) [reply deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py:149]: My fault, sorry 🙊

**Pull request merged**